src/CrawlAbstractLvl/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (C) 2020-2021 QuasarApp.
+# Distributed under the lgplv3 software license, see the accompanying
+# Everyone is permitted to copy and distribute verbatim copies
+# of this license document, but changing it is not allowed.
+#
+
+cmake_minimum_required(VERSION 3.14)
+set(CURRENT_PROJECT "AbstractLvl")
+
+add_definitions(-DSnakeProject_LIBRARY)
```

Comment:
you must change the default library define to CRAWL_ABSTRACT_LEVEL_LIBRARY define
and do not forget change in in your global.h file